### PR TITLE
Assign encounters to individual UI

### PR DESCRIPTION
**Pull request by brmscheiner**
image

### CODE REVIEW

File: src/pages/merge/MergeIndividuals.jsx [hunk 1]
```diff
@@ -93,11 +81,17 @@ export default function MergeIndividuals() {
...
-              key={individual?.guid || i}
-              data={individual}
-              mergeConflicts={mergeConflicts}
+              key={individualGuid}
+              individualGuid={individualGuid}
+              showSex={mergeConflicts?.sex}
+              setIndividualData={individualData =>
+                setIndividuals({
+                  ...individuals,
+                  [individualGuid]: individualData,
```
Comment (Atticus29) [inline src/pages/merge/MergeIndividuals.jsx:92]: I keep forgetting that you can do that! Cool.

File: src/pages/merge/MergeIndividuals.jsx [hunk 2]
```diff
@@ -169,8 +163,8 @@ export default function MergeIndividuals() {
               );
 
               mergeIndividuals({
-                targetIndividualGuid: targetIndividualData?.guid,
-                fromIndividualGuids: [fromIndividualData?.guid],
+                targetIndividualGuid: individualGuids?.[0],
```
Comment (Atticus29) [inline src/pages/merge/MergeIndividuals.jsx:166]: Wait, you can do that?!

File: src/pages/sighting/encounters/Encounters.jsx [hunk 3]
```diff
@@ -171,6 +171,7 @@ export default function Encounters({
       />
 
       <ManuallyAssignModal
+        sightingGuid={sightingId}
```
Comment (Atticus29) [inline src/pages/sighting/encounters/Encounters.jsx:174]: Might be worth it if you're in here anyway to change sightingId to sightingGuid as well...

File: src/pages/sighting/encounters/ManuallyAssignModal.jsx [hunk 4]
```diff
@@ -77,12 +80,11 @@ export default function ManuallyAssignModal({
             disabled={!selectedIndividualId}
             loading={loading}
             onClick={async () => {
-              const result = await addEncounterToIndividual(
-                selectedIndividualId,
-                encounterId,
-              );
-              console.log(result);
-              /* pick up here when success happens */
+              await assignEncounters({
```
Comment (Atticus29) [inline src/pages/sighting/encounters/ManuallyAssignModal.jsx:83]: No need for the async/await here if you don't need it for anything downstream.

**Review by Atticus29 — APPROVED** [review]
Just minor stuff! Looks good!

File: src/pages/merge/MergeIndividuals.jsx (hunk 2 above)
Comment (brmscheiner) [reply src/pages/merge/MergeIndividuals.jsx:166]: yep! I just learned it the other day.

File: src/pages/sighting/encounters/Encounters.jsx (hunk 3 above)
Comment (brmscheiner) [reply src/pages/sighting/encounters/Encounters.jsx:174]: I was gonna but i realized i have multiple PRs interacting with this same file and didnt wanna create a bunch of merge conflicts!

File: src/pages/createIndividual/CreateIndividual.jsx [hunk 5]
```diff
@@ -92,7 +92,12 @@ export default function CreateIndividual() {
           <Text variant="h4" id="CREATE_INDIVIDUAL" />
         </Grid>
         <Grid item>
-          <Text><Text variant="subtitle2" id="CREATE_INDIVIDUAL_INSTRUCTIONS" /></Text>
+          <Text>
```
[the remark this answers is not in the document]
Comment (brmscheiner) [reply src/pages/createIndividual/CreateIndividual.jsx:95]: just a mistake! good find

**Pull request merged**